src/types/error_responses.ts

```diff
@@ -0,0 +1,11 @@
+import { Registration } from './registration.types';
+
+export interface Problem {
+  title: string;
+  status: number;
+  requestId: string;
+}
+
+export interface DeletedRegistrationProblem extends Problem {
+  resource: Registration | undefined;
```

Comment:
Blir det samme, men ser bittelitt ryddigere ut
```suggestion
  resource?: Registration;
```